application.js

```diff
@@ -8,47 +8,53 @@
 function runChallenges() {
 
   // Ex 1. Read the content of the email input
-  var email = null;  // TODO: replace null, keep the email variable.
+  var email = "dleuliette@gmail.com";  // TODO: replace null, keep the email variable.
 
 
   // Ex 2. Fill the content of the email input with your email
-
-
+  $('#email').val("simon.ansel@outlook.com");
 
   // Ex 3. Replace the email hint (next to the input) with 'This is my email now'
   //       The text should be emphasized using a <strong> tag
-
+  $('#email-hint').html("<strong>This is my email now</strong>");
 
   // Ex 4. Add the .blue CSS class to the table header cells (th elements)
-
+  $("th").addClass('blue');
 
 
   // Ex 5. Count the number of table body rows there are (team count!)
-  var team_count = 0;  // TODO: replace 0, keep the team_count variable.
-
+  // TODO: replace 0, keep the team_count variable.
+  var team_count = $('tbody tr').length;
 
 
   // Ex 6. Say there is a 15th team added to the table.
   //       Add a row at the bottom, this new team should have zero points.
 
-
+  $('.table tbody:last').append("<tr><td>15</td><td>Wimereux</td><td>0</td>tr>");
```

Comment:
Why you didn't use the variable `team_count` ? like this

    var row = $('<tr>').html('<td>' + (team_count  + 1) + '</td><td>New team</td><td>0</td></tr>');
    $('tbody').append(row);